Hw 2

bananananacat opened the pull request.

bananananacat committed:
Create garbage

bananananacat committed:
Add files via upload

bananananacat committed:
Delete notebooks/02-homework.ipynb

bananananacat committed:
Add files via upload

bananananacat committed:
Delete notebooks/garbage

svkov commented on notebooks/02-homework.ipynb:
```diff
@@ -0,0 +1,341 @@
+    },
+    {
+     "data": {
+      "image/png": "iVBORw0KGgoAAAANSUhEUgAACXIAAANVCAYAAADf0ZTrAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMiwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8pXeV/AAAACXBIWXMAAA9hAAAPYQGoP6dpAAEAAElEQVR4nOzdd5xcZb0/8M/p07fXNEJJCCEUQULASwihCARUQPyBSlFBBUVEBIMKBCmXonIFsXKJCGK5olIUgQvYCBjhUgIhAiGkbsm2md2ZOfX5/XHOnJ3Zkp2QZFs+b15hds6cOXPmTNmdZz7P9ysJIQSIiIiIiIiIiIiIiIiIiIiIiIhozMhjvQNERERERERERERERERERERERES ...
```
Уже отвечал про это на лекции. Задача не в том, чтобы выдвинуть гипотезу и проверить ее, а в том, чтобы посмотреть на все данные и найти там инсайты. Нужно построить больше графиков и убедиться, что там больше нет зависимостей.

svkov replied:
Ну и спойлер - несколько важных зависимостей ты упустил)

svkov reviewed (changes requested):
Ставлю 0.1+
Доделай медиум

bananananacat committed:
Исправил ошибки

Доделал Medium(стало в 2 раза больше графиков), увидел новые зависимости, улучшил хард(добавив еще фильтрацию по возрасту)

bananananacat committed:
Доисправлял ошибки

Графиков стало в 2 раза больше, добавил описание, увидел незамеченные ранее зависимости, улучшил хард, добавив фильтрацию по возрасту.

svkov reviewed (approved):
Ок
